DTS-STN/Alpha-Site
# Feature/#215 experiments data cms

# Description

[Pull data from Strapi to display experiments](https://trello.com/c/BAQmDqRY/215-215-pull-data-from-strapi-to-display-experiments)

This task covers using getStaticProps in the Experiments page to fetch available experiments from the CMS api and use that data to render experiments on the page. Working with translation.

## Acceptance Criteria

Experiments shown in the Experiments page should be fetched from the CMS api and should be fully translated.

## Test Instructions

1. npm run dev
2. navigate to Experiments page
3. see experiments shown in the grid
4. view the fetch under Network in broswer devtools

## Help Requested

- Anything you think could be improved or that I missed

## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
